### PR TITLE
request remove

### PR DESCRIPTION
**Pull request by goodbiruk**
ready to review!

### CODE REVIEW

File: src/app/pages/request-detail/request-detail.component.ts [hunk 1]
```diff
@@ -84,7 +84,15 @@ export class RequestDetailComponent implements OnInit {
   }
 
   confirmRemove() {
-    this.removal_content = "Removal triggered."
+    this.removal_content = "Removal triggered.";
+
+    this.requestService.removeRequest(this.request_id).subscribe(response => {
+      if(response.success) {
+
```
Comment (practicalit) [inline src/app/pages/request-detail/request-detail.component.ts:91]: what is this? why do we have it as if and else wthout doing anything?

Comment (goodbiruk) [reply src/app/pages/request-detail/request-detail.component.ts:91]: this.requestService.removeRequest(this.request_id).subscribe(response => {
      if(response.success) {
        this.removal_content = "Request detail successfully deleted.";
      } else {
        this.removal_content = "Failed while deteting request detail.";

Comment (goodbiruk) [reply src/app/pages/request-detail/request-detail.component.ts:91]: this hat i'm trying to do

File: src/app/pages/home/home.component.ts [hunk 2]
```diff
@@ -52,7 +52,8 @@ export class HomeComponent implements OnInit {
       this.requestService.listRequest().subscribe(
         response => {
           if (response.success) {
-            this.requests = response.data;
+            debugger;
```
Comment (practicalit) [inline src/app/pages/home/home.component.ts:55]: we don't need this. please remove it

Comment (goodbiruk) [reply src/app/pages/home/home.component.ts:55]: copy that!

**Pull request merged**